src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -78,10 +85,17 @@ export class CalciteDropdownItem {
     });
   }
 
+  connectedCallback() {
+    let iconPosition = ["start", "end"];
+    if (this.icon !== null && !iconPosition.includes(this.iconPosition))
+      this.iconPosition = "start";
+  }
+
   render() {
     const dir = getElementDir(this.el);
     const theme = getElementTheme(this.el);
     const scale = getElementProp(this.el, "scale", "m");
+    const iconScale = scale === "s" || scale === "m" ? "s" : "m";
```

Comment:
Did you consider using a slot for the icon instead? Then a user can size them independently

Reply:
Yeah... currently anything can just be added to the `<slot>` where the text goes, so that's already possible. 

Setting it up in a .. more controlled way like this allows us ensure consistent styling and scale of icons out of the box, because if we don't we'll end up with a bunch of apps that look like this: 

<img width="205" alt="Screen Shot 2020-02-06 at 9 56 25 AM" src="https://user-images.githubusercontent.com/4733155/73964450-0fdc2980-48c7-11ea-83e9-48404fc6dbc8.png">

Not sure if this is a component that would be somewhere someone might want to use other sizes, I think we should try to encourage Popover use for more ... custom things? That said, it's always possible for folks to just put  `<calcite-icon>` in the slot and style it themselves with CSS if they want to, although it's not really documented for the above reason